src/ImageSharp.Web/Commands/CommandParser.cs

```diff
@@ -36,20 +36,23 @@ public CommandParser(IEnumerable<ICommandConverter> converters)
     /// </typeparam>
     /// <returns>The converted instance or the default.</returns>
     [MethodImpl(MethodImplOptions.AggressiveInlining)]
-    public T ParseValue<T>(string value, CultureInfo culture)
+    public bool TryParseValue<T>(string? value, CultureInfo culture, [NotNullWhen(true)] out T? parsedValue)
     {
         DebugGuard.NotNull(culture, nameof(culture));
 
         Type type = typeof(T);
-        ICommandConverter converter = Array.Find(this.converters, x => x.Type.Equals(type));
+        ICommandConverter? converter = Array.Find(this.converters, x => x.Type.Equals(type));
 
         if (converter != null)
         {
-            return ((ICommandConverter<T>)converter).ConvertFrom(
+            _ = ((ICommandConverter<T>)converter).TryConvertFrom(
```

Comment:
Shouldn't a `null` `parsedValue` be a false return?

It's been years since I've looked at this code.

Reply:
here we just handle it a few lines later with a null check. Didn't want to change everything